client/src/services/UserService.js

```diff
@@ -168,6 +168,35 @@ export class UserService {
 	setDefaultWallet = async (index) => {
 		this.currentWalletIndex = parseInt(index, 10);
 	}
+
+	generateWallets = async (total) => {
```

Comment:
Please update UT for this `generateWallets`. This should cover:
- `total` is 0
- `total` value is larger than allocated wallet number (10)
- `total` is less than existing wallet (???)

Reply:
Just updated it :D